packages/markup-frontstage/declarations.d.ts

```diff
@@ -0,0 +1,8 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
was this file actually needed? I recall you adding this to try and get tests working when importing scss files in components

Reply:
It is needed, I wonder why would you want to remove it?